packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -686,16 +752,54 @@ export class RemoteMediaManager extends EventsScope {
         );
       }
     }
-    // todo: screenshare (SPARK-377812)
 
     this.mediaRequestManagers.video.commit();
   }
 
+  private updateScreenShareVideoRemoteMediaObject() {
+    if (this.config.screenShare.video) {
+      this.invalidateCurrentRemoteMedia({
+        audio: false,
+        video: false,
+        screenShareAudio: false,
+        screenShareVideo: true,
+        commit: false,
+      });
+
+      this.media.screenShare.video = undefined;
+
+      if (this.currentLayout?.screenShareVideo) {
+        if (this.config.screenShare.video) {
+          // we create a group of 1, because for screen share we need to use the "active speaker" policy
```

Comment:
there might be more than one screenshare in a conference and you want the active-est? just checking I understood.

Reply:
no, it's just because how the WCME layer works. You have to use the "active speaker" policy to get the current screen share. Otherwise if you used "receiver selected" you would have to keep asking for a new CSI every time the presenter changes. RemoteMedia is currently designed only to work with "receiver selected" policy, while RemoteMediaGroup is meant for "active speaker", so I'm using RemoteMediaGroup internally for screen share (but when we emit the VideoLayoutChanged event to the app we give just a RemoteMedia instance for screen share to the app)